Mathlib/Data/Matrix/Basic.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2018 Ellen Arlt. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Ellen Arlt, Blair Shi, Sean Leather, Mario Carneiro, Johan Commelin, Lu-Ming Zhang
 -/
+import Mathlib.Algebra.Algebra.Opposite
```

Comment:
Is this necessary / deliberate?

Reply:
Same question for `Mathlib/Algebra/Algebra/Operations.lean`

Reply:
Yes, the opposite algebra is needed by this file and is no longer provided by `Algebra.Algebra.Basic`. I'll edit the commit message.